HiP-DataStore/Controllers/MediaController.cs

```diff
@@ -235,10 +209,12 @@ public IActionResult GetFileById(int id)
             if (!ModelState.IsValid)
                 return BadRequest(ModelState);
 
+            // TODO: This method requires rework - we should not query the read model here
```

Comment:
Maybe we can remove this lines and write something like this.
if (!_entityIndex.Exists(ResourceType.Media, id))
        return NotFound();

And for the old file removing we can add field to MediaTypeInfo class e.g. string FilePath. And then checking and deliting old file using only MediaIndex